### PR TITLE
fix: added check for the aviable models before init resent models (issue #488)

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Deployment status: success

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Deployment status: success

File: src/constants/errors.ts [hunk 1]
```diff
@@ -20,4 +20,6 @@ export const errorsMessages = {
     'Error happened during file request. Please try again later.',
   errorGettingUserFileBucket:
     'Error happened during getting file user bucket. Please reload the page to being able to load files.',
+  noModelsAviable:
```
Comment (mikitabut) [inline src/constants/errors.ts]: typo in error name

File: src/store/models/models.reducers.ts [hunk 2]
```diff
@@ -71,12 +71,17 @@ export const modelsSlice = createSlice({
         defaultModelId: string | undefined;
       }>,
     ) => {
+      const isDefaultModelAviable = state.models.some(
```
Comment (mikitabut) [inline src/store/models/models.reducers.ts]: typo in var name

File: src/constants/errors.ts (hunk 1 above)
Comment (denys-kolomiitsev) [reply src/constants/errors.ts]: Typo fixed

File: src/store/models/models.reducers.ts (hunk 2 above)
Comment (denys-kolomiitsev) [reply src/store/models/models.reducers.ts]: Typo fixed

**Review by mikitabut — APPROVED** [review]

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**